app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
-  end
+  # def current_user
+  #   User.first
+  # end
```

Comment:
- [ ] You should remove this method and implement a new method provided by `Devise` to configure Devise, please kindly add it and make sure the `Devise` is setup correctly using the [getting started link](https://github.com/heartcombo/devise) 🙏🏾 
- [ ] The requirements say users should register on the app using email and password, you can add other things like "name, bio, image URL" unfortunately, I was able to access the app pages without login or signup. Kindly fix this issue.
- [ ] Also make sure that the app work correctly, after user login or signup he can create a post, add a comment, and like a post 🙃 😉

Reply:
All the task are done as per your review.